website/README.md

```diff
@@ -29,10 +32,36 @@ npm run build
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
 
-## Deployment
+## Localization
 
-```console
-GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
```

Comment:
prehistoric deployment instruction, and inaccurate too (I bet it's straight-up generated from docusaurus template). Now it's automated in CI and firebase. No manual instruction is needed anymore.